README.md

```diff
@@ -1,11 +1,283 @@
+
+##### Manually Hooking up Previous or Next Videos <a name="t34"></a>
+
+There are many legitimate app UX circumstance, that can dictate the dynamicness of a video player – meaning, that not every app design will simply be setup to operate off fixed playlists or lists of videos. As such, the Player can be modified on the fly to dynamically handle what video is played when the previous or next buttons are tapped. This example tutorial has sample code that shows you precisely how to do this. However, be judicious with the usage of this behavior, and make sure it matches a natural flow of content for the user.
+
+### Tutorial 4 – Error Handling in the SDK <a name="t4"></a>
+**insert tutorial link here**
+
+This tutorial sample shows you how to handle various different types of errors that can occur when using the OMSDK and how to catch and identify them. How you handle these in your app is up to you. The SDK is designed to either return a valid SDK or Player instance otherwise it returns an error. There is no middle ground. If you don’t get a valid instance, you should look at the error result instead to determine why. This section describes some common issues.
+
+##### SDK Initialization Errors <a name="t41"></a>
+
+For various reasons, the SDK may fail to initialize. The most common reason for this, is you’re trying to use the OMSDK without first having [onboarded your app’s bundle ID](##Onboarding\ your\ Apps\ for\ SDK\ Authentication). In this case, you’ll get an error that looks like something like this:
+```
+{
+  "error": "Not found - com.company.ungregisteredapp"
+} 
+```
+
+##### Player Initialization Errors <a name="t42"></a>
+
+For various reasons, the `Player` may fail to initialize. 
+
+##### Restricted Videos <a name="t43"></a>
+
+Videos can be restricted for playback in two very distinct ways. The first is geo restricted content. The second is device restricted content. If you’re attempting to initialize a `Player` with content that’s restricted against your device or geolocation, that content is automatically filtered out. Only valid, playable video IDs are accepted, and have their metadata pulled into the `Player` instance. If you end up with no `Player` instance, it’s because there are no valid video IDs for it to operate on. So, you get an error this this effect.
+
+## Specific Notes for tvOS Apps <a name="tvos"></a>
+
+The OMSDK supports tvOS with the same source framework as iOS. Besides not having a finger to tap on the screen, the biggest difference for tvOS is that you cannot use the SDK’s default video player controls UX. Similarly, you cannot create your own custom video player controls UX. You must play videos using the standard built-in tvOS video player, with remote-control support.
```

Comment:
That's not true - about custom tvOS AVPlayer implementation. You can have custom UI. Saying that you are not allowed make us incompetent in this question.

Reply:
right!  strike this "Similarly, you cannot create your own custom video player controls UX. "